src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -12,12 +12,17 @@
 
 @if (_trace is { } trace)
 {
-    <div class="trace-detail-layout">
-        <div class="trace-detail-header page-header">
-            <h1 style="display:inline-block">@GetResourceName(trace.FirstSpan.Source): @trace.FirstSpan.Name</h1>
-            <span class="trace-id">@OtlpHelpers.ToShortenedId(trace.TraceId)</span>
-        </div>
-        <FluentToolbar Orientation="Orientation.Horizontal">
+    <AspirePageContentLayout
```

Comment:
The "View trace details" button shows content that says "Filters" at the top. There are no filters.

![view-trace-details](https://github.com/dotnet/aspire/assets/303201/2dc5281f-fec0-4877-8319-e8c194b4753d)

Reply:
Fixed